avro-serializer/src/test/java/io/confluent/kafka/serializers/KafkaAvroSerializerTest.java

```diff
@@ -232,4 +238,41 @@ public void testInvalidInput() {
       // this is expected
     }
   }
+
+  @Test
+  public void testEnableAutoSchemaRegistrationWhenSchemaVersionExists() {
```

Comment:
I don't think think this exercises anything that testKafkaAvroSerializer doesn't already exercise?